tools/Wildlife/wildlife_detection_classification.xml

```diff
@@ -0,0 +1,111 @@
+<tool id="wildlife_megadetector_huggingface" name="Wildlife Detection" version="0.1.0+galaxy0" profile="24.2">
+    <description>
+        Detect and classify wildlife species in images and videos using MegaDetector and Hugging Face models.
+    </description>
+    <requirements>
+        <container type="docker">
+            quay.io/arthur_barreau/wild_life:latest
+            <!--quay.io/bgruening/cameratrap:1.2.1-->
+        </container>
+    </requirements>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    mkdir -p output_files &&
+
+    #set element_ids = []
+    #for f in $data_dir
+        #set _ = $element_ids.append($f.element_identifier)
+    #end for
+    #set element_ids_str = " ".join($element_ids)
+
+    python "$__tool_directory__/main.py" \
+            "${model_name}" \
+            "${classifier_model}" \
+            "${config}" \
+            "${type_mapping}" \
+            "${boxing_mode}" \
+            "${data_dir}" \
+            "${detection_threshold}" \
+            "${stride}" \
+            "${images_max}" \
+            "./outputs" \
+            $element_ids_str
+    ]]>
+    </command>
+    <inputs>
+        <param name="model_name" type="text" value="facebook/dinov2-large" label="Hugging Face backbone model name" optional="false" help="Specify the pretrained model from Hugging Face to use as the image feature extractor" />
```

Comment:
can this be a select box?

Or a type=text with options, at least? I'm worried that users don't know which models to choose.